pkg/objstore/objstore.go

```diff
@@ -219,6 +229,12 @@ func BucketWithMetrics(name string, b Bucket, r prometheus.Registerer) Bucket {
 	}
 	if r != nil {
 		r.MustRegister(bkt.ops, bkt.opsFailures, bkt.opsDuration, bkt.lastSuccessfullUploadTime)
+		for _, op := range []string{iterOp, sizeOp, getOp, getRangeOp, existsOp, uploadOp, deleteOp} {
+			bkt.ops.WithLabelValues(op)
+			bkt.opsFailures.WithLabelValues(op)
+			bkt.opsDuration.WithLabelValues(op)
+		}
+		bkt.lastSuccessfullUploadTime.WithLabelValues(b.Name())
```

Comment:
I know you haven't introduced this but maybe we should fix this typo as well while changing this so that it would be `lastSuccessfulUploadTime`? :P

Reply:
:+1: